File: WebApp/Models/MM/MM_PR_DTBAS.cs
using System;
using System.Collections.Generic;

namespace WebApp.Models
{
    public class MM_PR_DTBAS : JCLib.Mvc.BaseModel
    {
        public string MAT_CLSID { get; set; }
        public string M_STOREID { get; set; }
        public string DATEBAS { get; set; }
        public string BEGINDATE { get; set; }
        public string ENDDATE { get; set; }
        public string SUMDATE { get; set; }
        public string MTHBAS { get; set; }
        public string LASTDELI_MTH { get; set; }
        public string LASTDELI_DT { get; set; }
        public string MMPRDTBAS_SEQ { get; set; }
        public string CREATE_TIME { get; set; }
        public string CREATE_USER { get; set; }
        public string UPDATE_TIME { get; set; }
        public string UPDATE_USER { get; set; }
        public string UPDATE_IP { get; set; }
    }
}

File: WebApp/Scripts/app/model/MM_PR_DTBAS.js
Ext.define('WEBAPP.model.MM_PR_DTBAS', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'MAT_CLSID', type: 'string' },
        { name: 'M_STOREID', type: 'string' },
        { name: 'DATEBAS', type: 'string' },
        { name: 'BEGINDATE', type: 'string' },
        { name: 'ENDDATE', type: 'string' },
        { name: 'SUMDATE', type: 'string' },
        { name: 'MTHBAS', type: 'string' },
        { name: 'LASTDELI_MTH', type: 'string' },
        { name: 'LASTDELI_DT', type: 'string' },
        { name: 'MMPRDTBAS_SEQ', type: 'string' },
        { name: 'CREATE_TIME', type: 'string' },
        { name: 'CREATE_USER', type: 'string' },
        { name: 'UPDATE_TIME', type: 'string' },
        { name: 'UPDATE_USER', type: 'string' }
    ]
});
